### PR TITLE
ACMS-702: Further improvement needed for drush command 'acms:config-reset'

### PR DESCRIPTION
**Pull request by chandan-singh7929**
**Motivation**
ACMS-702

Fixes #NNN

**Proposed changes**

Improvement in drush command 
```
drush acms:config-reset
```

**Testing steps**

Run `acms:config-reset` drush command with variations and check all are working fine.
Interactive:
```
drush acms:config-reset
drush acms:config-reset acquia_cms_article,acquia_cms_person,acquia_cms_place
drush acms:config-reset acquia_cms_article acquia_cms_person acquia_cms_place --scope=all
```
Non-interactive:
```
drush acms:config-reset acquia_cms_article acquia_cms_person acquia_cms_place --scope=all -y
```
**Merge requirements**
- [ ] _Major change_, _Minor change_, _Bug_, _Enhancement_, and/or _Chore_ label applied
- [x] Manual testing by a reviewer

### CODE REVIEW

File: modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php [hunk 1]
```diff
@@ -185,9 +185,11 @@ public function __construct(
    * @aliases acr
    * @usage acms:config-reset
    *   Reset the configuration to the default.
-   * @usage acms:config-reset acquia_cms_article acquia_cms_common --scope=all
+   * @usage acms:config-reset acquia_cms_article acquia_cms_article acquia_cms_person --scope=all
```
Comment (vishalkhode1) [inline modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php:188]: @chandan-singh7929 Argument: _acquia_cms_article_ is added twice. Possibly we need to remove one. Ex:
```suggestion
   * @usage acms:config-reset acquia_cms_article acquia_cms_person --scope=all
```

**Review by vishalkhode1 — APPROVED** [review]

File: modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php (hunk 1 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php]: @vishalkhode1 - Thanks, updated.

File: modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php [hunk 2]
```diff
@@ -544,4 +618,31 @@ private function hasValidPackage(array $packages): bool {
     return TRUE;
   }
 
+  /**
+   * Execute site studio rebuild after Acquia CMS config reset.
+   *
+   * @hook post-command acms:config-reset
+   */
+  public function postCommand($result, CommandData $commandData) {
```
Comment (vishalkhode1) [inline modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php:626]: @chandan-singh7929 We are doing the similar thing in _drush/Commands/SiteInstallCommands.php_ i.e. Cohesion rebuild. Please see, if we can reuse code and avoid code duplication. #750

**Comment by chandan-singh7929** [comment]
@vishalkhode1 - I have refactor the code for site studio rebuild and created function in acms utility services so that it can be used at both places. without duplicating the code.

File: modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php (hunk 2 above)
Comment (chandan-singh7929) [reply modules/acquia_cms_common/src/Commands/AcmsConfigImportCommands.php]: Done, as we have refactor the code this increase the test scope of the ticket. Now we have to check site install and see that site studio build is happening after site-install.

**Pull request merged**